index.html

```diff
@@ -7,10 +7,99 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="content">
+        <header>
+          <div class="logo">
+            <img class="logo-img" src="image/cake1.jpeg" alt="logo" />
+          </div>
+          <div>
+            <img src="image/hamburger.jpg" class="hamburger-menu" />
+          </div>
+          <p><i><b>The best cakes in town</b></i></p>
+          <p><i><b>delivered to your door</b></i></p>
+        </header>
+        <nav>
+          <ul class="navbar">
+            <li class="nav-item">
+              <a class="nav-link" href="#">HOME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">CAKES</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link">ORDERING</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Delivery</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">ABOUT</a>
+            </li>
+          </ul>
+        </nav>
+        <main>
+          <div class="intro">
```

Comment:
Don't forget sematic tags, replace 'div's with more meaningful alternatives

Reply:
Thank you, I  will change something.